MapboxNavigation/InstructionsBannerView.swift

```diff
@@ -114,19 +114,6 @@ open class BaseInstructionsBannerView: UIControl {
     public func update(for currentLegProgress: RouteLegProgress) {
```

Comment:
What does this function do now? We should figure out a good path forward for developers who use this function currently.